pandas/tests/frame/test_indexing.py

```diff
@@ -1925,6 +1925,50 @@ def test_getitem_ix_float_duplicates(self):
         expect = df.iloc[[1, -1], 0]
         assert_series_equal(df.loc[0.2, 'a'], expect)
 
+    def test_getitem_ix_categorical_index(self):
```

Comment:
I would move these tests to ``tests/indexing/tests_categoricaly.py`` (I just reorged this recently)